src/containers/main-page/preview/Languages.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  Grid, Typography, Paper, Box,
+} from '@mui/material';
+import CircleIcon from '@mui/icons-material/Circle';
+import MenuBookIcon from '@mui/icons-material/MenuBook';
+import { useIsFetching } from 'react-query';
+import { useEffect } from 'react';
+import { useUserFromDb } from '../cv-form/components/fields/personal-information/lib/query-hooks';
+import RatingLanguage from './RatingLanguage';
+
+const Languages = function (): JSX.Element {
+  const { data, refetch } = useUserFromDb();
+  const isFetching = useIsFetching('db-user');
+  useEffect(() => {
+    if (isFetching) { refetch(); }
+  }, [refetch, isFetching]);
+  return (
+    <Paper
+      elevation={2}
+      sx={{
+        maxWidth: '100%',
+        padding: '1.6rem',
+        margin: '18px 24px 0',
+        borderRadius: '0.2rem',
+      }}
+    >
+      <Grid
+        container
+        spacing={1}
```

Comment:
if there are two props please place them in one line
`<Grid container spacing={1} />`